### PR TITLE
Dịch "gan_vn" - Phần 3

### LINKED ISSUE #2729: Dịch "gan_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_generative-adversarial-networks/gan_vn.md#L148-L388

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by rootonchair**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2729 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 1]
```diff
@@ -164,7 +164,10 @@ We will illustrate what happens if we use GANs to build the world's most ineffic
 Let us get started.
 -->
 
-*dịch đoạn phía trên*
+Rất nhiều ứng dụng của GAN là liên quan tới hình ảnh.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:167]:
```suggestion
Rất nhiều ứng dụng của GAN liên quan tới hình ảnh.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 2]
```diff
@@ -164,7 +164,10 @@ We will illustrate what happens if we use GANs to build the world's most ineffic
...
+Nhằm mục đích ví dụ, chúng ta sẽ bắt đầu với việc khớp một phân phối đơn giản trước.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:168]:
```suggestion
Để ví dụ, chúng ta sẽ bắt đầu với việc khớp một phân phối đơn giản trước.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 3]
```diff
@@ -212,7 +215,8 @@ Let us see what we got.
 This should be a Gaussian shifted in some rather arbitrary way with mean $b$ and covariance matrix $A^TA$.
 -->
 
-*dịch đoạn phía trên*
+Dựa trên những gì ta có.
+Dữ liệu này có thể là một phân phối Gauss được dịch chuyển theo những cách khá bất thường với trung bình $b$ và ma trận hiệp phương sai $A^TA$.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:219]:
```suggestion
Dựa vào đoạn mã trên,
dữ liệu này là một phân phối Gauss được dịch chuyển theo những cách khá bất thường với trung bình $b$ và ma trận hiệp phương sai $A^TA$.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 4]
```diff
@@ -242,7 +246,9 @@ This is since we will be driving that linear network with a Gaussian data genera
 Hence, it literally only needs to learn the parameters to fake things perfectly.
 -->
 
-*dịch đoạn phía trên*
+Mạng bộ sinh sẽ là một mạng đơn giản nhất có thể - một mô hình tuyến tính đơn tầng.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:249]:
```suggestion
Bộ sinh sẽ là một mạng đơn giản nhất có thể - một mô hình tuyến tính đơn tầng.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 5]
```diff
@@ -242,7 +246,9 @@ This is since we will be driving that linear network with a Gaussian data genera
...
+Đó là vì chúng ta sẽ sử dụng mạng tuyến tính đó cùng với bộ sinh dữ liệu từ phân phối Gauss.
+Nên nó chỉ cần học những tham số của phân phối này để đóng giả một cách hoàn hảo.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:251]:
```suggestion
Nên nó chỉ cần học những tham số của phân phối này để làm giả một cách hoàn hảo.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 6]
```diff
@@ -260,15 +266,16 @@ net_G = nn.Sequential(nn.Linear(2, 2))
...
 
 
 <!--
 For the discriminator we will be a bit more discriminating: 
 we will use an MLP with 3 layers to make things a bit more interesting.
 -->
 
-*dịch đoạn phía trên*
+Đối với bộ phân biệt, nó sẽ hơi khác một chút:
+ta sẽ sử dụng một MLP 3 tầng để khiến mọi thứ trở nên thú vị hơn một chút
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:278]:
```suggestion
ta sẽ sử dụng một MLP 3 tầng để khiến mọi thứ trở nên thú vị hơn một chút.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 7]
```diff
@@ -347,7 +354,8 @@ The generator is updated similarly.
 Here we reuse the cross-entropy loss but change the label of the fake data from $0$ to $1$.
 -->
 
-*dịch đoạn phía trên*
+Bộ sinh cũng được cập nhật theo một cách tương tự.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:357]:
```suggestion
Bộ sinh cũng được cập nhật theo cách tương tự.
```

**Review by cuongvng — COMMENTED** [review]
Thanks!

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 8]
```diff
@@ -535,7 +543,7 @@ Tên đầy đủ của các reviewer có thể được tìm thấy tại https
 * 
 
 <!-- Phần 3 -->
-* 
+* Phạm Hồng Vinh
```
Comment (rootonchair) [inline chapter_generative-adversarial-networks/gan_vn.md:546]:
```suggestion
* Phạm Hồng Vinh
* Nguyễn Văn Cường
```

**Review by cuongvng — APPROVED** [review]

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 9]
```diff
@@ -212,7 +215,8 @@ Let us see what we got.
 This should be a Gaussian shifted in some rather arbitrary way with mean $b$ and covariance matrix $A^TA$.
 -->
 
-*dịch đoạn phía trên*
+Dựa vào đoạn mã trên,
+dữ liệu này là một phân phối Gauss được dịch chuyển theo những cách khá bất thường với trung bình $b$ và ma trận hiệp phương sai $A^TA$.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/gan_vn.md:219]:
```suggestion
dữ liệu này là một phân phối Gauss được dịch chuyển một cách tùy ý với trung bình $b$ và ma trận hiệp phương sai $A^TA$.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 10]
```diff
@@ -242,7 +246,9 @@ This is since we will be driving that linear network with a Gaussian data genera
 Hence, it literally only needs to learn the parameters to fake things perfectly.
 -->
 
-*dịch đoạn phía trên*
+Bộ sinh sẽ là một mạng đơn giản nhất có thể - một mô hình tuyến tính đơn tầng.
+Đó là vì chúng ta sẽ sử dụng mạng tuyến tính đó cùng với bộ sinh dữ liệu từ phân phối Gauss.
+Nên nó chỉ cần học những tham số của phân phối này để làm giả một cách hoàn hảo.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/gan_vn.md:251]:
```suggestion
Nên nó chỉ cần học những tham số của phân phối này để làm giả dữ liệu một cách hoàn hảo.
```

**Review by minhduc0711 — APPROVED** [review]
thanks, mình thấy ổn rồi

**Review by duythanhvn — APPROVED** [review]
